gef-ui/server/utils/displayItems.js

```diff
@@ -104,17 +106,11 @@ const facilityItems = (exporterUrl) => [
     isCurrency: true,
   },
   {
-    label: 'Interest margin your bank will charge',
+    label: type === 0 ? 'Interest margin your bank will charge' : 'Risk margin your bank will charge',
```

Comment:
would be good to use facilityType = 'something' rather than numbers, it's ambiguous. Think this was mentioned in another PR, maybe being worked on somewhere else? Not a blocker for me.

Reply:
This will need to be on BE as well as FE. Will fix this in our next Story

Reply:
It would be good to use constants too

Reply:
Happy to correct this in the next story